suite-common/token-definitions/src/tokenDefinitionsUtils.ts

```diff
@@ -0,0 +1,60 @@
+import { NetworkSymbol, getNetworkFeatures } from '@suite-common/wallet-config';
+
+import {
+    DefinitionType,
+    SimpleTokenStructure,
+    TokenDefinitionsState,
+    TokenManagementAction,
+    TokenManagementStorage,
+} from './tokenDefinitionsTypes';
+
+export const caseContractAddressForNetwork = (
+    networkSymbol: NetworkSymbol,
+    contractAddress: string,
+) => (networkSymbol === 'sol' ? contractAddress : contractAddress.toLowerCase());
+
+export const isTokenDefinitionKnown = (
+    tokenDefinitions: SimpleTokenStructure | undefined,
+    networkSymbol: NetworkSymbol,
+    contractAddress: string,
+) =>
+    Array.isArray(tokenDefinitions)
+        ? tokenDefinitions?.includes(caseContractAddressForNetwork(networkSymbol, contractAddress))
+        : false;
+
+export const getSupportedDefinitionTypes = (networkSymbol: NetworkSymbol) => {
+    const isCoinDefinitionsEnabled = getNetworkFeatures(networkSymbol).includes('coin-definitions');
+    const isNftDefinitionsEnabled = getNetworkFeatures(networkSymbol).includes('nft-definitions');
+
+    return [
+        ...(isCoinDefinitionsEnabled ? [DefinitionType.COIN] : []),
+        ...(isNftDefinitionsEnabled ? [DefinitionType.NFT] : []),
+    ];
+};
+
+export const buildTokenDefinitionsFromStorage = (
```

Comment:
nit: better naming

Reply:
There is `buildHistoricRatesFromStorage` that does same similar stuff so I would leave it as it is